advocacy_docs/partner_docs/Cyberark/04-Configuratingcyberark.mdx

```diff
@@ -0,0 +1,44 @@
+---
+title: 'Configuration'
+description: 'Walkthrough on configuring the integration'
+---
+
+## Prerequisites
+
+Implementing CyberArk CPM (Central Policy Manager) with EDB Postgres Advanced Server requires the following components:
+
+## Configuring your EDB Postgres Advanced Server
+
+Following component is needed before integrating EDB Postgres Advanced Server with CyberArk CPM (Central Policy Manager):
+
+1. One running instance of EDB Postgres Advanced Server.
```

Comment:
If this is also for Postgres should we say something like "A running instance of EPAS or Postgresql"?

Reply:
Testing is done for EPAS only so mentioned EPAS here, same is the case with Connect CDC guids as well